config/scripts/lp-pull.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * @file
+ * Get the latest sanitized database for local development.
+ *
+ * This will only work if you have the creds set up in
+ * your .env file.
+ *
+ * If you don't have those creds, but would like to contribute
+ * to the backdropcms.org code base please request them on:
+ *
+ * Zulip: https://backdrop.zulipchat.com/#narrow/stream/218635-Backdrop
+ * 
+ * or file a new issue on 
+ * https://github.com/backdrop-ops/backdropcms.org/issues/
+ *
+ * requesting either a copy of the DB or access to creds.
+ *
+ * Allotting access to creds will be made on a case by case basis.
+ *
+ * If you have the creds make sure to set them in your .env file and
+ * do a `lando rebuild -y` to let lando load the env vars into the app.
+ */
+
+$user = getenv('LP_USER');
+$pass = getenv('LP_PASS');
+$db_url = getenv('LP_DB_URL');
+$files_url = getenv('LP_FILES_URL');
+
+if (!isset($user) || !isset($pass)) {
```

Comment:
I wanted to see this message displayed, so I purposefully didn't create an `.env` file and ran `lando pulldb`, but it didn't show this... Instead I got:
```bash
wget: missing URL
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
File /app/backdropcmsorg-latest-sanitized.sql.gz not found!
```

Reply:
let me see if including the URL vars in this helps.